src/OrbitGl/ThreadTrack.cpp

```diff
@@ -82,19 +82,11 @@ void ThreadTrack::InitializeNameAndLabel() {
 }
 
 const TimerInfo* ThreadTrack::GetLeft(const TimerInfo& timer_info) const {
-  if (timer_info.thread_id() == GetThreadId()) {
-    const TimerChain* chain = track_data_->GetChain(timer_info.depth());
-    if (chain != nullptr) return chain->GetElementBefore(timer_info);
-  }
-  return nullptr;
+  return scope_tree_.GetLeft(timer_info.depth() + 1, timer_info.start());
```

Comment:
Oh actually I think this is a problem, the depth in the TimerInfo will not be correct when mixing manually and dynamically instrumented scopes.

Reply:
We might have to add a `FindScopeNode` method in `ScopeTree` to find the node based on the start and end times.

Reply:
Added FindScopeNode and am using the depth based on the scope node and it is now working when mixing manually and dynamically instrumented scopes.